include/dali/c_api.h

```diff
@@ -17,6 +17,7 @@
 
 #include <cuda_runtime_api.h>
 #include <inttypes.h>
+#include <stdbool.h>
```

Comment:
Don't use the stdbool.h but rather convert the functions to use integers. No one says how the C `bool` typedef maps to C++'s bool type.

Reply:
Done